ls1intum/Artemis
# Upgrade to Angular 11

### Checklist

### Motivation and Context
Supporting Angular 11 seems to be easy :-)
No major changes were needed.

### Steps for Testing
Click through the most important functionality of the Artemis app and check if something is broken or not
